changelog/Microsoft.AAD.Reporting.json

```
@@ -1733,6 +1733,40 @@
       "CreatedDateTime": "2017-03-01T00:00:00",
       "WorkloadArea": "Reports",
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "b211329e-698e-4909-aa0c-4b967538bc13",
+          "ApiChange": "Resource",
+          "ChangedApiName": "membershipOutlierInsight",
+          "ChangeType": "Addition",
+          "Description": "Added the [membershipOutlierInsight](https://docs.microsoft.com/en-us/graph/api/resources/membershipOutlierInsight?view=graph-rest-beta) resource type. Use this type to configure insight settings based on the user's last sign-in date and time into a tenant or application.",
```

Is this complex type really configurable? I thought it's read-only.

If confirmed to be read-only, just remove the second sentence from the changelog description.
```suggestion
          "Description": "Added the [membershipOutlierInsight](https://docs.microsoft.com/en-us/graph/api/resources/membershipOutlierInsight?view=graph-rest-beta) resource type. Use this type to configure insight settings based on the user's last sign-in date and time into a tenant or application.",
```